Code/Tools/W3DView/Utils.cpp

```diff
@@ -944,7 +944,7 @@ Copy_File
 
 	// Perform the copy operation!
 	if (allow_copy) {
-		retval = (::CopyFile (existing_filename, new_filename, false) == true);
```

Comment:
why change this to `TRUE`?

Reply:
I think because [`CopyFile`](https://learn.microsoft.com/en-us/windows/win32/api/winbase/nf-winbase-copyfile) returns a `BOOL` which is a `int` typedef.

Reply:
Unfortunately the API in question is a win32 api so not under our control. I've updated the PR to be consistent across all the changes now.